src/readpst.c

```diff
@@ -1151,6 +1152,31 @@ void write_embedded_message(FILE* f_output, pst_item_attach* attach, char *bound
     DEBUG_RET();
 }
 
+/**
+ * Backslash-escape quotes in the given string.
+ */
+char *escape_quotes(char *inp) {
```

Comment:
This is an unrelated change to add Outlook-friendly content-disposition headers for attachments with utf8 filenames.  In particular, Outlook doesn't support the `filename*=utf-8''...` format (all the attachments are displayed with "unknown filename" placeholders), but it does handle actual utf-8 characters in the `filename` parameter, which is technically invalid but generally supported (and ultimately more user-friendly than the correct alternative of stripping out all non-ASCII characters). I'm personally not thrilled with producing invalid output, but it is Outlook... 

Nuix sidesteps this by specifying the filename as a RFC2047-encoded "name" parameter in the "Content-Type" header, which Outlook also supports.  I'm not sure that's standard, though.